### PR TITLE
Submission of second homework assignment

### PR DESCRIPTION
**Pull request by sebnaran**
Hello Professor Niemeyer,

I finally got travis to work! I did look at how other students figured out how to do it, I hope that is okay...

I want to apologize for leaving town without mentioning anything to you, things have been happening very quickly and I have not reacted to everything in time. I want to reassure you that I am committed to the course and thank you for the extension to complete this task.

### CODE REVIEW

**Comment by kyleniemeyer** [comment]
@sebnaran great!

@tommy-moore22 I believe this is for you to review

File: Functions.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import math
+##These functions relate to the initiation of the code and mesh
```
Comment (tommy-moore22) [inline Functions.py:3]: You should use the docstring notation from the notes for the definition of your functions comments

Comment (tommy-moore22) [reply Functions.py:3]: This should be used for all of your function comments

File: Test_Functions.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+def test_StaggeredSpatialMeshInternal():
+    #Here we will test that the meshing function can divide the interval [-1,1] 
+    #into five pieces. 
+    N = 5
+    xplus, xminus, dx = StaggeredSpatialMesh(N)    
+    txplus = np.array([-1, -0.6, -0.2, 0.2, 0.6, 1])
+    txminus = np.array([-0.8, -0.4, 0, 0.4, 0.8])
+    assert np.linalg.norm(txplus-xplus) < 1e-3
+    assert np.linalg.norm(txminus-xminus) < 1e-3
+test_StaggeredSpatialMeshInternal()
```
Comment (tommy-moore22) [inline Test_Functions.py:13]: You don't need to call the functions since they will be tested using pytest

**Review by tommy-moore22 — APPROVED** [review]
See comments within the code. Good work!

**Comment by tommy-moore22** [comment]
I do not have permission to merge your pull request, but feel free to do so.

**Pull request merged**